packages/record-data/types/@ember/runloop/index.d.ts

```diff
@@ -0,0 +1,2 @@
+
+export const run: any;
```

Comment:
O_o seems bad?

Reply:
would you want to try to type backburner :D

Reply:
haha, I did 😉

Reply:
We do this specifically because we're reaching in to avoid triggering the autorun assertion, left a comment